certbot-dns-digitalocean/certbot_dns_digitalocean/dns_digitalocean_test.py

```diff
@@ -50,7 +50,7 @@ def test_cleanup(self):
 
 
 class DigitalOceanClientTest(unittest.TestCase):
-    id = 1
+    _id = 1
```

Comment:
I *think* mypy didn't like this because we were re-assigning python's built-in `id` function, but I'm not sure. In any case, renaming this to `_id` seems to make mypy happy and not break the test.